web_tree_dynamic_colored_field/static/description/index.html

```diff
@@ -0,0 +1,533 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion

```
XML declarations in HTML module descriptions are deprecated since Odoo 17

Reply:
I think this is autogenerated by `   pre-commit run --all-files`

I'll look how to fix it (of course I can manually, but maybe you have hints )

Reply:
> I think this is autogenerated by ` pre-commit run --all-files`
> 
> I'll look how to fix it (of course I can manually, but maybe you have hints )

I think you're right, it is autogenerated. In that case, nevermind. If you find it necessary, you can take it out manually so that the warning doesn't appear. Thanks in advance!

Reply:
now accepted your suggestion -- and sometime autogen will handle this as well, I guess

Reply:
Thanks!